packages/p2p-v2/src/components/Modals/AdErrorTooltipModal/AdErrorTooltipModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+import { Button, Text } from '@deriv-com/ui';
+import Modal from 'react-modal';
+import { ADVERT_TYPE, ERROR_CODES } from '@/constants';
+import { AdRateError } from '@/pages/my-ads/components';
+import { customStyles } from '../helpers';
+import './AdErrorTooltipModal.scss';
```

Comment:
these imports arent auto sorted 👀

Reply:
yeah. had to remove eslint extension from vscode due to some issue.. now got the errors after reinstalling.. fixing it now..